content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,225 @@ The tables below show changes made to the installation options used to customize
-| `gateways.istio-ingressgateway.ports.name` | `https-kiali` |  |
-| `gateways.istio-ingressgateway.ports.name` | `https-prometheus` |  |
-| `gateways.istio-ingressgateway.ports.name` | `https-grafana` |  |
-| `gateways.istio-ingressgateway.ports.targetPort` | `15032` |  |
-| `gateways.istio-ingressgateway.ports.name` | `https-tracing` |  |
-| `gateways.istio-ingressgateway.ports.targetPort` | `15443` |  |
-| `gateways.istio-ingressgateway.ports.name` | `tls` |  |
-| `gateways.istio-ingressgateway.ports.targetPort` | `15020` |  |
-| `gateways.istio-ingressgateway.ports.name` | `status-port` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `15011` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-pilot-grpc-tls` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `15004` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-mixer-grpc-tls` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `8060` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-citadel-grpc-tls` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `853` |  |
-| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-dns-tls` |  |
-| `gateways.istio-ingressgateway.env.ISTIO_META_ROUTER_MODE` | `"sni-dnat"` | `A gateway with this mode ensures that pilot generates an additionalset of clusters for internal services but without Istio mTLS, toenable cross cluster routing.` |
-| `gateways.istio-ingressgateway.nodeSelector` | `{}` |  |
-| `gateways.istio-egressgateway.autoscaleEnabled` | `true` |  |
-| `gateways.istio-egressgateway.cpu.targetAverageUtilization` | `80` |  |
-| `gateways.istio-egressgateway.podAnnotations` | `{}` |  |
-| `gateways.istio-egressgateway.ports.targetPort` | `15443` |  |
-| `gateways.istio-egressgateway.ports.name` | `tls` |  |
-| `gateways.istio-egressgateway.env.ISTIO_META_ROUTER_MODE` | `"sni-dnat"` |  |
-| `gateways.istio-egressgateway.nodeSelector` | `{}` |  |
-| `gateways.istio-ilbgateway.autoscaleEnabled` | `true` |  |
-| `gateways.istio-ilbgateway.cpu.targetAverageUtilization` | `80` |  |
-| `gateways.istio-ilbgateway.podAnnotations` | `{}` |  |
-| `gateways.istio-ilbgateway.nodeSelector` | `{}` |  |
+| `gateways.istio-ingressgateway.sds.resources.requests.cpu` | `100m` |  |
+| `gateways.istio-ingressgateway.sds.resources.requests.memory` | `128Mi` |  |
+| `gateways.istio-ingressgateway.sds.resources.limits.cpu` | `2000m` |  |
+| `gateways.istio-ingressgateway.sds.resources.limits.memory` | `1024Mi` |  |
+| `gateways.istio-ingressgateway.resources.requests.cpu` | `100m` |  |
+| `gateways.istio-ingressgateway.resources.requests.memory` | `128Mi` |  |
+| `gateways.istio-ingressgateway.resources.limits.cpu` | `2000m` |  |
+| `gateways.istio-ingressgateway.resources.limits.memory` | `1024Mi` |  |
+| `gateways.istio-ingressgateway.applicationPorts` | `""` |  |
```

Comment:
@howardjohn Extracting the descriptions correctly has been a tricky thing to do. In the code, I do a lot of checks to filter out descriptions to make sure we get the right descriptions for the right configuration option. For example, in some of the values.yaml, old configuration options have been commented out and if they appear right before the next configuration option, without those filters in place, it will consider those commented out options as comments for it. So I think if the comment line contains the format: <something>: <something>, I just ignore it. The thinking was it was better to not have a description vs putting the wrong description for a configuration option. I will go ahead and verify this one more time and see if there is anything I can do to get this description. @ostromart was working on creating a more robust system to work as a schema for these configuration options which is why we did not go down the route of cleaning up all these values.yaml to remove the commented out configuration options.

Reply:
Ok sounds good! just wasnted sure why it was happening, the `:` makes sense. Thanks

Reply:
Sounds good.